features/step_definitions/requests.js

```diff
@@ -55,4 +55,4 @@ module.exports = function () {
         assert.equal(this.processError.process, binary);
```

Comment:
I didn't remove "large request" test which I added recently - it will be useful to avoid regression as was introduced by Boost.Beast migration.

Reply:
Agreed.